.github/workflows/ci.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           - validate-go-mod
           - validate-headers
           - validate-docs
+          - validate-fmt
```

Comment:
FWIW; there's a golangci-lint linter for `gofumpt`; https://golangci-lint.run/usage/linters/#gofumpt

Reply:
nice, makes things even simpler